File: core/src/main/java/com/graqr/threshr/model/redsky/product/ProductVendor.java
package com.graqr.threshr.model.redsky.product;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record ProductVendor(
        String id,
        @JsonProperty("vendor_name")
        String vendorName) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/ProductVendor.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class ProductVendor(val id: String, @param:JsonProperty("vendor_name") val vendorName: String) {}
